compilers/concrete-compiler/compiler/lib/Support/CompilerEngine.cpp

```diff
@@ -264,6 +265,15 @@ CompilerEngine::compile(llvm::SourceMgr &sm, Target target, OptionalLib lib) {
 
   auto dataflowParallelize =
       options.autoParallelize || options.dataflowParallelize;
+  if (options.optimizerConfig.strategy == optimizer::Strategy::DAG_MULTI) {
+    // FIXME: DF is not currently compatible with multi-parameters as
+    // the generation of dataflow tasks obfuscates the code before the
+    // analysis can be done. Until this is fixed we cannot allow both.
+    dataflowParallelize = false;
+    warnx("WARNING: dataflow parallelization is not compatible with the "
```

Comment:
Isn't this warning only happen if `dataflowParallelize` is `true` :thinking:

Reply:
Yes - why would we warn when it's false?
Or do you instead mean we should only warn if true - which indeed would be better. Fixed this way

Reply:
That's what I meant, perfect!